data/human/ships.txt

```diff
@@ -2299,15 +2299,15 @@ ship "Mammoth"
 	attributes
 		category "Utility"
 		"cost" 18800000
-		"shields" 7800
+		"shields" 6200
 		"hull" 7400
 		"required crew" 18
 		"bunks" 30
-		"mass" 688
-		"drag" 10.5
+		"mass" 680
```

Comment:
I don't think we need to decrease mass..?

Reply:
Derpy wants us to be at 0 endings for numbers like this. Either 680 or 690.

Reply:
Not sure why we have that standard but alright. Since it was 688 so probably should round up to 690.

Reply:
Your typical human ship has mass or cargo or outfit or weapon or engine space in multiples of 5, but I don't think jerry-rigged pirate ships need necessarily follow that trend, although the Modified Argosy actually does, so maybe we change that instead of having every pirate variant behave as a typical shipyard-manufactured ship.

Reply:
Ah okay. ~~Grumble grumble my Gegno stats.~~